addons/portal/models/portal_mixin.py

```diff
@@ -30,3 +38,10 @@ def get_share_url(self):
             params.update(self.partner_id.signup_get_auth_param()[self.partner_id.id])
 
         return '/mail/view?' + url_encode(params)
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu=False):
```

Comment:
looks suspicious

Reply:
You look suspicious.

Reply:
More seriously. Couldn't we override postprocess_and_fields method of ir.ui.view in order to add the "share" action, along with create, delete and edit ? That way we could consider it as a "main" action of the record along with those other actions.

Reply:
@tde-banana-odoo , @mart-e ,  Override 'postprocess_and_fields'  in commit: https://github.com/odoo/odoo/pull/22142/commits/b92df84f7069deb290b7fe87b687fc216c90986e
 but each time we need to check 'portal.mixin' is inherited or not is it ok for you? Thanks

Reply:
It seems better integrated to me. @mart-e what do you think ?